frontend/src/metabase/query_builder/components/QueryVisualization.jsx

```diff
@@ -96,9 +101,7 @@ export function VisualizationRunningState({ className = "" }) {
       )}
     >
       <LoadingSpinner />
-      <h2 className="Loading-message text-brand text-uppercase my3">
-        {message}
-      </h2>
+      <h2 className="text-brand text-uppercase my3">{message}</h2>
```

Comment:
2 questions.
1. Is `Loading-message` unused?
2. Should you convert the rest 3 global classes to CSS modules (CS.xxx)?